tests/test_visitors/test_ast/test_complexity/test_overuses/test_overused_string.py

```diff
@@ -178,7 +181,7 @@ def test_string_overuse(
         string_value.replace('"', '') or "''",
         default_options.max_string_usages,
     )
-    assert_violation_location(visitor)
+    assert_error_location(visitor, expected_location)
```

Comment:
Can we just create a single test with known location? Like `x = 'abc'` and have it as a constant? Like `(1, 4)`? Right now it seems too complex.